### DISCUSSION TITLE
Failed to invoke the method，IP does not exist

### DISCUSSION THREAD

**Discussion by xhrg** [apache/dubbo#14410] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

nacos-server：2.0.3
nacos-client：1.4.2.jar

In the K8S environment, the service consumer reported an error and called the IP of a non-existent service provider, but the service provider did indeed experience a restart. In the container of the service consumer, the cache file of the Naming directory shows that all the IPs inside are correct, but Dubbo calls the old IP that does not exist before.

k8s环境，服务消费者报错，调用了一个不存在的服务提供者的ip，服务提供者的确发生了重启。在服务消费者的容器里面，naming目录的缓存文件看，里面的ip都是正确的，但是dubbo却去调用不存在的之前的老ip。

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
What is your Dubbo version
